PHILOSOPHY.md

```diff
@@ -0,0 +1,158 @@
+## Design goals
+
+The design goals for Tendermint (and the SDK and related libraries) are:
+
+ * Simplicity and Legibility
+ * Parallel performance, namely ability to utilize multicore architecture
+ * Ability to evolve the codebase bug-free
```

Comment:
Is this even possible? I mean there're ways to reduce the surface area, but nobody? succeeded in growing a codebase 100% bug-free.